### PR TITLE
[Cordova] Upgrade the value of minSdkVersion parameter from 14(default) to 16 when building cordova tests.

### PR DESCRIPTION
**Pull request by BruceDai**
BUG=https://crosswalk-project.org/jira/browse/CTS-1831

### CODE REVIEW

**Comment by BruceDai** [comment]
@haoxli , PTAL

**Comment by BruceDai** [comment]
@haoxli PTAL

File: usecase/usecase-cordova-android-tests/samples/CordovaPackage/res/test.py [hunk 1]
```diff
@@ -126,9 +129,10 @@
     elif BUILD_PARAMETERS.pkgarch == "arm64":
         pack_arch_type = "armv7 --xwalk64bit"
     pack_arch_opt = "-- --gradleArg=-PcdvBuildArch=%s" % pack_arch_type
+    min_sdk_version = "--minSdkVersion=16"
 
 
-os.system("cordova build android %s" % pack_arch_opt)
+os.system("cordova build android %s %s" % (pack_arch_opt, min_sdk_version))
 os.system("cordova run android %s" % pack_arch_opt)
```
Comment (haoxli) [inline usecase/usecase-cordova-android-tests/samples/CordovaPackage/res/test.py]: Update   to `os.system("cordova run android  %s %s" % (pack_arch_opt, min_sdk_version))`

**Comment by haoxli** [comment]
`cordova run android` also need "-- --minSdkVersion=16"

**Comment by haoxli** [comment]
lgtm

**Pull request merged**